test/big.js

```diff
@@ -504,6 +504,29 @@ exports.jacobi = function () {
     assert.equal(b1.jacobi(b2), 1);
 };
 
+exports.base2 = function () {
+  assert.equal(bignum('-1234734').toString(2), '-100101101011100101110');
+  assert.equal(bignum('-2').toString(2), '-10');
```

Comment:
Is this the correct behavior?  Should this be the bytes for a signed int32/int64 in the case of negative values?